provisioning/roles/mailcatcher/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+- name: ensure needed packages are installed
+  apt: pkg={{ item }} state=latest
+  sudo: yes
+  with_items:
+    - ruby
+    - ruby-dev
```

Comment:
There's a `ruby` role, maybe add a dependency on it instead of installing ruby here.